src/components/Data/Collections/Create.vue

```diff
@@ -1,11 +1,86 @@
 <template>
   <div class="wrapper">
-    CollectionCreate
+    <headline>
+      Create a collection in <i>{{$route.params.index}}</i>
```

Comment:
I think we have to merge this PR before
I will merge the conflits after on the #37 an re check existences of $route in views